main/settings.py

```diff
@@ -217,7 +217,6 @@
         "BACKEND": "django.template.backends.django.DjangoTemplates",
         "DIRS": [
             BASE_DIR + "/templates/",
-            BASE_DIR + "/frontends/mit-learn/build",
```

Comment:
Is this needed for anything?

Reply:
It was necessary when django served the JS